src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -812,7 +765,7 @@ public void onPlayerChat(AsyncPlayerChatEvent event) {
 		event.setFormat(format);
 	}
 
-	@EventHandler(priority = EventPriority.NORMAL)
+	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)

```

Comment:
I feel like with this eventHandler changed, the lines just below there do the exact same thing.   
if (event.isCancelled())
-				return;

Unless there's just some super special circumstance where shockwave needs to be checked even if the event is cancelled, in that case you should probably re-add the other two lines that are removed in that method.